lib/plugins/aws/deploy/compile/events/apiGateway/lib/methods.js

```diff
@@ -6,6 +6,10 @@ const _ = require('lodash');
 module.exports = {
   compileMethods() {
     _.forEach(this.serverless.service.functions, (functionObject, functionName) => {
+      if (!functionObject.events) {
```

Comment:
IMHO I think it would be better / and easier to follow if the forEach is wrapped inside the if like this:

```
if (functionObject.events) {
   functionObject.events.forEach()
}
```

rather than doing a return like this

```
if (!functionObject.events) {
  return;
}
```

This also applies for the if clauses below.